teambit.bvm/install/installWithPnpm.ts

```diff
@@ -0,0 +1,79 @@
+import fs from 'fs';
+import path from 'path';
+import { getConfig, type Config } from '@pnpm/config';
+import { streamParser } from '@pnpm/logger';
+import { initDefaultReporter } from '@pnpm/default-reporter';
+import { install } from '@pnpm/plugin-commands-installation';
+import { readWantedLockfile } from '@pnpm/lockfile.fs';
+
+export async function installWithPnpm(fetch, version: string, dest: string) {
+  await fetchLockfile(fetch, version, dest);
+  await createPackageJsonFile(dest);
+
+  const cliOptions = {
+    argv: [],
+    dir: dest,
+    registry: 'https://node-registry.bit.cloud/',
```

Comment:
make configuration for this with bvm config, in case the user have its own registry that proxy to bit registry